src/main/java/picard/analysis/directed/RnaSeqMetricsCollector.java

```diff
@@ -60,12 +60,15 @@ public RnaSeqMetricsCollector(final Set<MetricAccumulationLevel> accumulationLev
         return new PerUnitRnaSeqMetricsCollector(sample, library, readGroup, ribosomalInitialValue);
     }
 
-    public static OverlapDetector<Interval> makeOverlapDetector(final File samFile, final SAMFileHeader header, final File ribosomalIntervalsFile) {
+    public static OverlapDetector<Interval> makeOverlapDetector(final File samFile, final SAMFileHeader header, final File ribosomalIntervalsFile, final double rrnaFragmentPercentage) {
 
         OverlapDetector<Interval> ribosomalSequenceOverlapDetector = new OverlapDetector<Interval>(0, 0);
         if (ribosomalIntervalsFile != null) {
 
             final IntervalList ribosomalIntervals = IntervalList.fromFile(ribosomalIntervalsFile);
+            if ( ribosomalIntervals.size() == 0 && rrnaFragmentPercentage == 0.0) {
+                throw new PicardException("The RIBOSOMAL_INTERVALS file, " + ribosomalIntervalsFile.getAbsolutePath() + " must containn intervals if the rRNA fragment percentage = 0");
```

Comment:
I wasn't suggesting to pass in `rrnaFragmentPercentage` to this method (I don't think it is appropriate), but ihnstead log a warning if `ribosomalIntervals.size() == 0`.